src/Symfony/Component/ErrorRenderer/composer.json

```diff
@@ -29,6 +29,7 @@
     },
     "autoload": {
         "psr-4": { "Symfony\\Component\\ErrorRenderer\\": "" },
+        "classmap": [ "Resources/stubs/Exception/FlattenException.php" ],
```

Comment:
should be removed

Reply:
if we remove this classmap, the applications that currently throw the legacy FlattenException class will break, no?

Reply:
Nothing throws this class (it's not a real exception). The report is using a type hint, so that it should work, no?

Reply:
Sorry I meant creating the legacy FlattenException, I'm assuming other cases, such as custom exception listeners/datacollectors/tests/handlers, etc. who currently use legacy FlattenException, should we wait for reports?

Reply:
I mean, not as type-hint

Reply:
These will use the Debug component

Reply:
I talking about current projects where the Debug component will be automatically removed after update... :/ we want a smooth update?

Reply:
That projects that actually use the Debug component + `FlattenException::create()` will break after update: Debug component removed automatically + unknown class or alias or `::create()` method not found.

Reply:
By doing this, we guarantee at least the object's creation through the alias, but I still think we should re-introduce the `create()` method.

Reply:
We provide no guarantee at the dependency level: if ppl use Debug but have it transitively, it's accepted policy to require them to make it an explicit dep as of the next version (minor included)
Said another way: the BC policy covers the API, not the deps.

Reply:
Okay, let's get this class map off then, thanks